app/services/app_store_client.rb

```diff
@@ -39,6 +39,24 @@ def search(payload)
     process_response(response, url)
   end
 
+  def post_import_error(message)
+    url = "#{host}/v1/failed_imports"
+    response = self.class.post(url, **options(message))
+
+    case response.code
+    when 201
+      JSON.parse(response.body)
+    else
+      raise "Unexpected response from AppStore - status #{response.code} for '#{message[:id]}'"
```

Comment:
Something to be aware of for the future (no change needed here) is that we often swallow and ignore these "unexpected app store" errors, we should come up with a better plan.

Reply:
I'll add a note to the dev meeting board and resolve this when I have